packages/redux/src/raffles/actions/__tests__/__snapshots__/createRaffleParticipation.test.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`createRaffleParticipation() action creator should create the correct actions for when the create raffle participation procedure is successful: createRaffleParticipation success payload 1`] = `undefined`;
```

Comment:
Hmm, `undefined` doesn't seem like the correct actions, something might be wrong on your tests 🤔 (all snapshots are like this)